src-vue/src/utils/filter.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Implements a fuzzy filter
```

Comment:
While one can understand fuzzy filtering by looking at this implementation, I would like to have some explanations here (maybe including an external documentation link like Wikipedia).

Reply:
Done in 5ee7a99403f7b8ed6ef9a45ae6e5d1f635dde509